### PR TITLE
Feature/webumenia 789 filter by curator

### PR DESCRIPTION
**Pull request by Endut**
# Description

This adds functionality to filter catalog by curator ('contributor' in the db)
also reimplements the harvesting of this curator data to conform with @rastislav-chynoransky spice-harvester refactor

Fixes #789 (link to Jira or GitHub issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/Harvest/Mappers/ItemMapper.php [hunk 1]
```diff
@@ -182,6 +182,10 @@ public function mapFeatured() {
         return false;
     }
 
+    public function mapContributor(array $row) {
+        return $row['contributor'][0] ?: null;
```
Comment (igor-kamil) [inline app/Harvest/Mappers/ItemMapper.php]: this could easily lead into "undefined index" exception. better to use `isSet()` od `!empty()` in this case.

Comment (Endut) [reply app/Harvest/Mappers/ItemMapper.php]: hey @igor-kamil  think this is fine now, don't know how to tell github i've implemented the change

**Comment by Endut** [comment]
tests passing and ready for merge i think

**Comment by igor-kamil** [comment]
approved ✅ 
ready to merge

**Pull request closed without merging**